x/or/da/types/params.go

```diff
@@ -2,31 +2,52 @@ package types
 
 import (
 	"fmt"
+	"math"
 
 	"gopkg.in/yaml.v2"
 )
 
 const (
-	DefaultCCBatchMaxBytes  uint64 = 1000000
-	DefaultSCCBatchMaxBytes uint64 = 1000000
+	DefaultCCBatchMaxBytes         uint64 = 1000_000
+	DefaultSCCBatchMaxBytes        uint64 = 1000_000
+	DefaultMaxQueueTxSize          uint64 = math.MaxUint16
+	DefaultMinQueueTxGas           uint64 = 300000
+	DefaultQueueTxExpirationWindow uint64 = 600
+	DefaultFraudProofWindow        uint64 = 7 * 24 * 60 * 60 / 6
+	DefaultSequencerPublishWindow  uint64 = 600
```

Comment:
What is `DefaultSequencerPublishWindow`?

Reply:
It means block interval. So if sequencers haven't submitted a batch in 600 blocks, any node can submit a state commitment.